src/stores/WalletStore.ts

```diff
@@ -21,6 +21,7 @@ class WalletStore extends PersistableStore {
   walletsToNotifiedOfBeingDoxxed = {} as {
     [address: string]: boolean
   }
+  firstBadge = { minted: false, notified: false, twitted: false }
```

Comment:
Should go away if:
clicking the CTA, reload, and on next visit, wallet had badges